### DISCUSSION TITLE
BUFR file has incompelete tables!

### DISCUSSION THREAD

**Discussion by zfy68** [Unidata/netcdf-java#1031] [Questions and Answers]
Hello, This toolsUI-5.5.2.jar program is being updated, but B/D table configuration information does not seem to be updated. It does not support the template of the new report. 
see screenshot below：
![image](https://user-images.githubusercontent.com/37278360/173320668-a76f963f-b3a0-4933-8d8f-4ec0b34ec445.png)
![image](https://user-images.githubusercontent.com/37278360/173323083-31b1cbf2-bb51-4cd0-8374-aa4542ea785b.png)

**Comment by JohnLCaron** [comment]
Hi #zfy68, do you know what BUFR version it wants? If not, can you attach the faiing BUFR message?

**Reply by zfy68** [reply]
in the second screenshot ：
table = 0.35.0，it need 35 table，but not found！
